grace/evaluation/process.py

```diff
@@ -28,30 +30,68 @@ def update_graph_with_dummy_predictions(
     Notes
     -----
     - Modifies the graph in place.
-    - Only motifs with the same object identity will be linked.
+    - The 'uncertainty' parameter separates the distributions of predictions
+        for the respective classes, e.g. 0.5 will produce a distribution of
+        attribute values for TN between [0.0, 0.5) & TP between (0.5, 1.0]
+        which are very close, whilst 0.1 will force larger separation between
+        them, i.e. TN between [0.0, 0.1) & TP between (0.9, 1.0].
+    - Only motifs with the same object identity should be linked (TODO)
     """
 
```

Comment:
Seems like you should add a check that `node_uncertainty` and `edge_uncertainty` are in an acceptable range and raise an error if not.

Reply:
Now implemented!